.env

```
@@ -54,3 +54,11 @@ MAILER_DELIVERY_ADDRESS="admin@example.org"
 # DATABASE_URL="postgresql://app:!ChangeMe!@127.0.0.1:5432/app?serverVersion=14&charset=utf8"
 DATABASE_URL="mysql://mail:password@127.0.0.1:3306/mail?charset=utf8mb4"
 ###< doctrine/doctrine-bundle ###
+
+### Enable keycloak API ###
+# Set to `true` to enable keycloak API
+KEYCLOAK_API_ENABLED=false
+# Restrict access to this IP
+KEYCLOAK_API_IP="127.0.0.1"
```

```suggestion
KEYCLOAK_API_IP_ALLOWLIST="127.0.0.1"
```
Would name that more precisely.

---

Given that it only supports one IP and no list, I think `KEYCLOAK_API_IP` is better. Or maybe `KEYCLOAK_API_ALLOW_IP`?

---

We must support multiple IPs or a Subnet. Otherwise it is impossible to support container based hostings or KeyCloak with HA setup.

---

I changed this now to use the [ips access control option](https://symfony.com/doc/6.4/security/access_control.html#matching-access-control-by-ip). This way, subnets are supported.